test/turbo/redirect_to.test.js

```diff
@@ -145,4 +145,18 @@ describe("redirect_to", () => {
       assert.deepEqual(Turbo.visit.args[0], expected)
     })
   })
+
+  context("turbo_frame attribute", () => {
+    it("renders the frame attribute", async () => {
+      sinon.replace(window, "Turbo", { visit: sinon.fake(), frame: "modals" })
+
+      await executeStream(`<turbo-stream action="redirect_to" frame="modals" url="http://localhost:8080"></turbo-stream>`)
```

Comment:
The test is failing because it expects the `frame` attribute to be `turbo-frame`. I know it's confusing, because the `action` attribute right next to it is not the `turbo-action` attribute, but the `action` from the Turbo Stream which determines which actually it actually is.

```suggestion
      await executeStream(`<turbo-stream action="redirect_to" turbo-frame="modals" url="http://localhost:8080"></turbo-stream>`)
```